template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/BlackListedDownScaleRole.java

```diff
@@ -1,13 +1,14 @@
 package com.sequenceiq.cloudbreak.cmtemplate;
 
 import com.sequenceiq.cloudbreak.auth.altus.model.Entitlement;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.cruisecontrol.CruiseControlRoles;
 import com.sequenceiq.cloudbreak.common.type.Versioned;
 
 import java.util.Objects;
 import java.util.Optional;
 
 public enum BlackListedDownScaleRole implements EntitledForServiceScale {
-    KAFKA_BROKER(Entitlement.DATAHUB_STREAMING_SCALING),
+    KAFKA_BROKER(Entitlement.DATAHUB_STREAMING_SCALING, null, CruiseControlRoles.CRUISECONTROL),
```

Comment:
I think we should add the minimum CDH version too as in theory someone can have a pre-7.2.12 custom blueprint based cluster with Cruise Control.
In this case Cruise Control is there but the CSD doesn't have the decommission hook.